python/pyspark/mllib/classification.py

```diff
@@ -31,13 +31,13 @@
            'SVMModel', 'SVMWithSGD', 'NaiveBayesModel', 'NaiveBayes']
 
 
-class LinearBinaryClassificationModel(LinearModel):
+class LinearClassificationModel(LinearModel):
     """
     Represents a linear binary classification model that predicts to whether an
     example is positive (1.0) or negative (0.0).
```

Comment:
Update this doc please.  Also, can you please add a note that this is a private abstract class?  (That's not because of your PR, but just because we should make that clear here.)